costmap_converter/include/costmap_converter/costmap_converter_interface.h

```diff
@@ -139,6 +139,7 @@ class BaseCostmapToPolygons
     virtual ObstacleArrayConstPtr getObstacles()
     {
       ObstacleArrayPtr obstacles = std::make_shared<costmap_converter_msgs::msg::ObstacleArrayMsg>();
+      obstacles->header.frame_id = "odom";
```

Comment:
Either I'm missing something or it's very weird that costmap converter publishes obstacles an ObstacleArrayMsg without a frame_id

Reply:
definitely. Ideally, we'd get the frame id from the costmap, but this will do for now as well